bin/sirius-script-si-acquire-bpm-triggered-data.py

```diff
@@ -2,25 +2,26 @@
 """."""
 
 from siriuspy.devices import BunchbyBunch, SOFB, HLFOFB
-from apsuite.commisslib.measure_orbit_stability import \
-        OrbitAcquisition
+from apsuite.commisslib.meas_bpms_signals import AcqBPMsSignals
 from siriuspy.devices import BOPSRampStandbyHandler, BORFRampStandbyHandler
 import sys
+from datetime import datetime
 
 
 def configure_acquisition_params(orbacq):
     """."""
     params = orbacq.params
+    params.signals2acq = 'XY'
 
     # params.orbit_acq_rate = 'FAcq'
     # params.orbit_timeout = 100
 
-    params.orbit_acq_rate = 'TbT'
-    params.orbit_timeout = 60*4  # 3 minutes between top-up injections
+    params.acq_rate = 'TbT'
+    params.timeout = 60*4  # 3 minutes between top-up injections
 
-    params.orbit_nrpoints_before = 1_000
-    params.orbit_nrpoints_after = 10_000
-    params.orbit_acq_repeat = 0
+    params.nrpoints_before = 1_000
+    params.nrpoints_after = 10_000
```

Comment:
since this is a general script, I think the number of points and the acquisition rates should be optional inputs. See `argparse`  python module.

Reply:
Thanks for the suggestion. Done!